### ISSUE TITLE
Error updating without `--force`

### ISSUE THREAD

**Issue by maifeeulasad** [btxtiger/ngx-link-preview#22]
When tried installing, with this command: `npm i ngx-link-preview`. It failed saying:
```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: news-update@0.0.0
npm ERR! Found: tslib@2.3.1
npm ERR! node_modules/tslib
npm ERR!   tslib@"^2.1.0" from the root project
npm ERR!   tslib@"^2.1.0" from @angular/common@12.0.5
npm ERR!   node_modules/@angular/common
npm ERR!     @angular/common@"~12.0.5" from the root project
npm ERR!     peer @angular/common@">=6.0.0" from ngx-link-preview@1.1.6
npm ERR!     node_modules/ngx-link-preview
npm ERR!       ngx-link-preview@"*" from the root project
npm ERR!     1 more (@angular/platform-browser)
npm ERR!   4 more (@angular/core, zone.js, @angular/platform-browser, @angular/animations)
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer tslib@"^1.10.0" from ngx-link-preview@1.1.6
npm ERR! node_modules/ngx-link-preview
npm ERR!   ngx-link-preview@"*" from the root project
npm ERR! 
```
Then forcing it to installed worked:
```
npm i --force ngx-link-preview
```

**Comment by jonaaix** [comment]
@maifeeulasad does it also work using `--legacy-peer-deps` ?

**Comment by maifeeulasad** [comment]
Yes, @btxtiger that worked: https://github.com/maifeeulasad/news-update/tree/776b3dccf0f4f6658aab99abd23f8ddb35086ff7

**Comment by jonaaix** [comment]
I need to figure out what's necessary to use the new npm dependency tree. Until then installing with `--legacy-peer-deps` will be required

**Comment by jonaaix** [comment]
Should be fixed with 1.2.x
